examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Examples
+
+The directory contains simple examples for OTA updates
+
+## beta
+
+This is the example app that will replace the running OTA app. Typically, any new image should contain OTA update logic.
+This app should be built before any of the ota apps can use it. 
+It is better to build in release mode to minify the application. 
+
+Build: 
+```
+RUSTFLAGS="-C strip=symbols -C debuginfo=0" cargo build --example beta --release
+espflash save-image --chip <mcu_target> ./target/<mcu_target>/release/examples/beta ./beta.bin
+```
+
+## ota_from_flash
+
+The app contains the update in it's flash memory at compile time. The `beta.bin` image in the ESP32 app image format must be available at compile time.
+
+Build:
+```
+cargo build --example ota_from_flash --release
+```
+Flash and monitor:
+```
+espflash flash ./target/<mcu_target>/release/examples/ota_from_flash --partition_table ./examples/partitions_4MB.csv --monitor 
+```
+
+## partitions_4MB
+
+A partition table that splits the two OTA partitions evenly.
+
+## partitions_factory.csv
+
+The default partition table shipped with the ESP32. _Not suitable for OTA._
```

Comment:
Why is this partition table included, since it's not used?

Reply:
I included it as a reference. Dodgy things might happen if a user doesn't re-write their original partition table. 
For example, if you flash OTA alpha and OTA beta into partition slot ota1 and ota2 respectively. In this case, OTA alpha loads, writes OTA beta into the ota2 slot and selects that as the boot partition. 

When a user wants to flash a non-ota app into what they expect to be the factory partition, the ota2 slot could still be selected as the  boot partition, and the factory partition never opens :-(

Reply:
I don't think it's the responsibility of this repository to track the default partition table. If there are potential caveats (which are currently not even mentioned) I think it's far better to describe how to get out of the bad situation, and link to an online resource with the default partition table. Then we don't have to maintain it here